packages/tame-global-error-object/test/tameGlobalErrorObject.test.js

```diff
@@ -4,18 +4,44 @@ import tameGlobalErrorObject from '../src/main.js';
 
 const { test } = tap;
 
-test('tameGlobalErrorObject', t => {
-  t.plan(3);
+test('tameGlobalErrorObject - captureStackTrace', t => {
+  t.plan(7);
 
   const restore = captureGlobals('Error');
```

Comment:
Is this calling https://github.com/Agoric/SES-beta/blob/master/packages/test262-runner/src/utilities.js#L16 ?
Should it be more like
```js
captureGlobals({ captureGlobalObjectNames: 'Error' });
```
?

We also do this elsewhere.